robots/herb.srdf.xacro

```diff
@@ -192,27 +220,41 @@
   <!-- hand arguments should all default to false to make argument passing sane -->
   <!-- NOTE: if we use kinematically distinct hands from the BarrettHand
        we will need more fine-grained arguments -->
-  <xacro:arg name="left_hand" default="false"/>
-  <xacro:arg name="right_hand" default="false"/>
+  <xacro:arg name="left_none" default="false"/>
```

Comment:
Let's delete the `*_none` arguments and do some xacro logic below to infer when to use `xacro:empty_endeffector`.

Reply:
Actually, maybe we can do something like https://github.com/ros/xacro/issues/191#issuecomment-427333003.